app/src/renderer/components/staking/PageDelegates.vue

```diff
@@ -69,15 +69,14 @@ export default {
   data: () => ({
     query: '',
     sort: {
-      property: 'id',
-      order: 'asc',
+      property: 'shares',
+      order: 'desc',
       properties: [
-        // { id: 1, title: 'Keybase ID', value: 'keybaseID', initial: true },
-        { id: 1, title: 'Public Key', value: 'id', initial: true },
-        { id: 2, title: 'Country', value: 'country' },
-        { id: 3, title: 'Voting Power', value: 'voting_power' },
-        { id: 4, title: 'Bonded Power', value: 'shares' },
-        { id: 5, title: 'Commission', value: 'commission' }
+        { id: 1, title: 'Name', value: 'description.moniker' },
+        { id: 2, title: 'Public Key', value: 'id' },
+        { id: 3, title: 'Voting Power', value: 'shares', initial: true },
+        { id: 4, title: 'Bonded Atoms', value: 'voting_power' },
```

Comment:
why not...?
```
{ id: 3, title: 'Voting Power', value: 'voting_power', initial: true },
{ id: 4, title: 'Bonded Atoms', value: 'bonded_atoms' },
```

Reply:
It makes more sense to call the percentage of power "Voting Power", since that's how it's been referred to outside of the code. And what's called 'voting_power' in the code isn't necessarily voting power, it's more accurate to refer to it as the number of bonded tokens.

Reply:
> percentage of power "Voting Power"

is percentage of power === voting power / bonded atoms?

> 'voting_power' in the code isn't necessarily voting power

😕  why not?

i'm probably just misunderstanding something...